### PR TITLE
Styled buttons

### LINKED ISSUE #26: Style buttons
Now the buttons are default style, style them to look better

### PR DESCRIPTION
**Pull request by kushaangowda**
styled the buttons

### CODE REVIEW

**Comment by indremak** [comment]
image
It looks like this on Chrome. @kushaangowda Could you fix "new game" button?

File: style.css [hunk 1]
```diff
@@ -112,19 +121,52 @@ button {
...
 }
 
+
 #game-over {
-  justify-content: center;
-  align-items: center;
-  width: 100%;
   position: absolute;
-  top: 200px;
+  top: 45%;
```
Comment (indremak) [inline style.css]: With these changes 'game over' isn't centered on smaller screens. Could you please fix this? 
image

**Comment by kushaangowda** [comment]
I have fixed new game btn, can u please check now?
*Will fix 'Game Over!'

File: style.css (hunk 1 above)
Comment (kushaangowda) [reply style.css]: sure

**Comment by indremak** [comment]
@kushaangowda yes, the button looks good now 🙂

**Comment by kushaangowda** [comment]
@indremak, I have fixed 'Game Over', can u please check?

**Comment by indremak** [comment]
@kushaangowda I suggest leave `display: flex` and `justify-content: center`, set `top: 350px`, because now scroll bar appears in x-axis.

**Comment by kushaangowda** [comment]
have changed diplay to flex and set top:350px
*Will fix the scroll bar

**Comment by kushaangowda** [comment]
@indremak, I have fixed it. If there is any other issue, please let me know

**Comment by indremak** [comment]
@kushaangowda Everything works, thank you for fixing bugs 🙂

**Pull request merged**